modules/home-environment.nix

```diff
@@ -164,13 +164,13 @@ in
     };
 
     home.sessionVariableSetter = mkOption {
-      default = "bash";
-      type = types.enum [ "pam" "bash" ];
+      default = "shell";
+      type = types.enum [ "pam" "shell" ];
       example = "pam";
       description = ''
         Identifies the module that should set the session variables.
         </para><para>
-        If "bash" is set then <varname>config.bash.enable</varname>
+        If "shell" is set then <varname>config.bash.enable</varname>
         must also be enabled.
```

Comment:
I intended to make this change when I submit the PR for the fish module. Right now the only supported shell is bash so I left it as-is.